### PR TITLE
feat: bitget adapter

### PR DESCRIPTION
**Pull request by tobikuhlmann**
## Description

This PR adds a Bitget exchange adapter

## Other changes

No

## Tested

- Tested locally with BRL config (https://github.com/celo-org/celo-monorepo/pull/10194/files)
- added a Bitget test file, all tests pass

## Related issues

- Fixes [#164](https://github.com/mento-protocol/mento-general/issues/164) [#181](https://github.com/mento-protocol/mento-general/issues/181)

## Backwards compatibility

Fully

### CODE REVIEW

File: test/exchange_adapters/bitget.test.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import BigNumber from 'bignumber.js'
+import { baseLogger } from '../../src/default_config'
+import { ExchangeAdapterConfig } from '../../src/exchange_adapters/base'
+import { BitgetAdapter } from '../../src/exchange_adapters/bitget'
+import { Exchange, ExternalCurrency } from '../../src/utils'
+
+describe('GeminiAdapter', () => {
```
Comment (philbow61) [inline test/exchange_adapters/bitget.test.ts:7]: BitgetAdapter ?

File: test/exchange_adapters/bitget.test.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      data: {
+        base_coin: 'BTC',
+        status: 'online',
+        symbol: 'btcbrl_SPBL',
+      },
+      msg: 'success',
+      requestTime: '0',
+    }
+
+    it("returns false when status isn't 'open'", async () => {
```
Comment (philbow61) [inline test/exchange_adapters/bitget.test.ts:88]: online?

**Review by philbow61 — CHANGES_REQUESTED** [review]

**Comment by tobikuhlmann** [comment]
thanks @philbow61, changes committed

**Review by philbow61 — APPROVED** [review]
LGTM 
- seems like the linter is not happy yet :( 
- try to run yarn lint and yarn prettify

File: src/exchange_adapters/bitget.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
+import { Exchange } from '../utils'
+import { BaseExchangeAdapter, ExchangeDataType, Ticker, Trade } from './base'
+
+export class BitgetAdapter extends BaseExchangeAdapter {
+  baseApiUrl = 'https://api.bitget.com/api'
+
+  readonly _exchangeName = Exchange.BITGET
+  // bitget.com cert
```
Comment (ninocomputer) [inline src/exchange_adapters/bitget.ts:8]: can you pls make this consistent so use the second license

**Review by ninocomputer — CHANGES_REQUESTED** [review]

File: src/exchange_adapters/bitget.ts (hunk 3 above)
Comment (tobikuhlmann) [reply src/exchange_adapters/bitget.ts]: done

**Review by ninocomputer — APPROVED** [review]

**Pull request merged**